tests/conftest.py

```diff
@@ -4,150 +4,185 @@
 # v. 2.0. If a copy of the MPL was not distributed with this file, You can
 # obtain one at http://mozilla.org/MPL/2.0/.
 import copy
+import json
 
 import pytest
 
 
-ATTACHMENT = {
-    "flags": [],
-    "attacher": "foobar@example.com",
-    "last_change_time": "2020-06-30T12:40:45Z",
-    "creation_time": "2020-06-30T12:40:45Z",
-    "is_patch": 0,
-    "description": "Testcase",
-    "creator_detail": {
-        "real_name": "Foo Bar (foobar)",
-        "email": "foobar@example.com",
+@pytest.fixture
+def attachment_data():
```

Comment:
For a lot of this, I've found it cleaner to create a fixtures directory under tests, store this kind of data as json or yaml, and read it as needed.

Reply:
I'm going to leave it for the time being but I will look into refactoring them later.